roles/kubernetes/control-plane/tasks/main.yml

```diff
@@ -66,3 +66,27 @@
 - name: Include kubelet client cert rotation fixes
   include_tasks: kubelet-fix-client-cert-rotation.yml
   when: kubelet_rotate_certificates
+
+- name: Install script to renew K8S control plane certificates
```

Comment:
why no `when: auto_renew_certificates` on this one ?

Reply:
ok seems like a good thing to have indeed, and it's not like this will take a lot of space.